### DISCUSSION TITLE
[Question]: How to change logo.svg?

### DISCUSSION THREAD

**Discussion by angpao** [danny-avila/LibreChat#4220] [Q&A]
### What is your question?

I used deploy-compose.yaml, I want to change logo of login.How to do?

### More Details

I used deploy-compose.yaml, I want to change logo of login.How to do?
<img width="327" alt="Screenshot 2567-09-24 at 16 34 22" src="https://github.com/user-attachments/assets/603d4de6-99c6-46f6-b08b-5a2decd4e77a">

### What is the main subject of your question?

UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by angpao** [comment]
If I will set deploy-compose.yaml to use "- ./images/logo.svg:/app/client/public/assets/logo.svg" , The logo will change. Is it possible?

**Comment by imran786110** [comment]
You need to deploy your own docker image instead of predefined docker image in docker-compose.yml file.

e.g. 
If you are done with your changes, e.g. logo changes or text changes, then run this command to create your own image.
`
docker build -t my-image .
`

It will take some time to complete.

Then simply edit your docker-compose.yml file at line 12 where image is given:
`
 image: my-image
`

When its complete simple run:
`docker-compose up -d`

It will now run from your local image.
